cfn-resources/network-peering/cmd/resource/resource.go

```diff
@@ -200,33 +186,35 @@ func Update(req handler.Request, prevModel *Model, currentModel *Model) (handler
 	}
 
 	peerID := *currentModel.Id
-	peerRequest := mongodbatlas.Peer{}
+	peerRequest := admin.BaseNetworkPeeringConnectionSettings{}
```

Comment:
wow, I didn't know about this, is this available for other methods to have a default request object?